spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -1,33 +1,33 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-org.springframework.cloud.gcp.autoconfigure.pubsub.GcpPubSubEmulatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.core.GcpContextAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.logging.StackdriverLoggingAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.pubsub.GcpPubSubAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.pubsub.GcpPubSubReactiveAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.spanner.GcpSpannerAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.spanner.GcpSpannerEmulatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.spanner.SpannerTransactionManagerAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.datastore.GcpDatastoreAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.firestore.GcpFirestoreAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.firestore.GcpFirestoreEmulatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.datastore.health.DatastoreHealthIndicatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.storage.GcpStorageAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.trace.StackdriverTraceAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.datastore.DatastoreRepositoriesAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.spanner.SpannerRepositoriesAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.security.IapAuthenticationAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.security.FirebaseAuthenticationAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.vision.CloudVisionAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.datastore.GcpDatastoreEmulatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.bigquery.GcpBigQueryAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.datastore.DatastoreTransactionManagerAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.firestore.FirestoreRepositoriesAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.pubsub.health.PubSubHealthIndicatorAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.metrics.GcpStackdriverMetricsAutoConfiguration
+com.google.cloud.spring.autoconfigure.pubsub.GcpPubSubEmulatorAutoConfiguration,\
```

Comment:
This seems like a big miss in my find/replace script, yet everything seems to work as expected. What does this file do?

Reply:
You didn't miss it. It's actually fine. I just had issues reapplying changes that I initially started on the old repo to here. Look at the overall diff.